Projects/Team2/Vlad Borisovets/UnitTestProject2/UnitTestProject/UnitTest1.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Firefox;
+using System.Threading;
+using NUnit.Framework;
+using OpenQA.Selenium.Interactions;
+
+namespace UnitTestProject
+{
+	public class UnitTest1
+	{
+		public IWebDriver driverChrome;
+		public IWebDriver driverFirefox;
+		
+		[OneTimeTearDown]
+		public void EndTest()
+		{
+			driverChrome.Quit();
+			driverFirefox.Quit();
+		}
+		
+		[Test]
+		[Parallelizable]
+		public void LoginChrome()
+		{
+			driverChrome = new ChromeDriver();
+			// prepare
+			AuthorizationPage.goToSite(driverChrome, "https://naurok.ua/");
+
+
+			// action
+			AuthorizationPage account = new AuthorizationPage("vladb707@gmail.com", "test_worked");
+			AuthorizationPage.Logout(driverChrome, account);
+
+
+			// verification      
+			string url = "https://naurok.ua/student/dashboard";
+			Assert.AreEqual(url, driverChrome.Url);
+		}
+		
+		[Test]
+		[Parallelizable]
```

Comment:
are these tests work in parallel? 
probably you need to use Parallelizable(Parallelescope.All)